hoot-services/src/main/java/hoot/services/controllers/export/ExportCommand.java

```diff
@@ -129,6 +129,14 @@ else if (translation.equalsIgnoreCase("translations/TDSv40.js")) {
             options.add("ogr.append.data=true");
         }
 
+        // OK. This is VERY UGLY and there has to be a better way to do this
+        // DNC uses some MIN INT values to signify NULL's If we don't change this, it throws an
+        // exception and dies.
+        if (params.getTranslation().equalsIgnoreCase("translations/DNC.js")) {
+            options.add("ogr.strict.checking=off");
```

Comment:
This is the command line option we need. Unfortunately, we can't just set it here. The Javascript is loaded after we need this option to be set so it has to go on the command line. Putting it here doesn't hurt,